### PR TITLE
bep(docs): flush out user roles a bit more

### PR DESCRIPTION
**Pull request by aramissennyeydd**
## Hey, I just made a Pull Request!

Possible small updates to the existing docs bep to better flush out what each role means.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: beps/0006-docs-personas-framework-portal/README.md [hunk 1]
```diff
@@ -134,55 +174,59 @@ None
...
-      - Getting Started
-      - Writing a permission policy
-      - Frontend integration
-      - Defining custom permission rules
-      - Using permissions in plugins
-    - Designing for Backstage
-    - ADRs
-    - Accessibility
-    - References
+    - Core Plugins
```
Comment (aramissennyeydd) [inline beps/0006-docs-personas-framework-portal/README.md:177]: I just moved the above into a "Core Plugins" block

File: beps/0006-docs-personas-framework-portal/README.md [hunk 2]
```diff
@@ -197,43 +241,47 @@ None
...
-      - Built in actions
-    - TechDocs
-      - Overview
-      - Getting Started
-      - Architecture
-        - Installing and configuring
-          - Using Cloud Storage for TechDocs generated files
-          - Configuring CI/CD to generate and publish TechDocs sites
-        - TechDocs CLI
+    - Core Plugins
```
Comment (aramissennyeydd) [inline beps/0006-docs-personas-framework-portal/README.md:244]: Same as above, moved the above into a Core Plugins section.

File: beps/0006-docs-personas-framework-portal/README.md [hunk 3]
```diff
@@ -79,6 +79,41 @@ The benefits of restructuring the documentation according to these ideas include
 - The Docs section of the microsite will be divided into two top-level sections: Framework and Developer Portal.
 - The structure of the Table of Contents will align with the outline proposed in https://github.com/backstage/backstage/issues/21946.
 
+### User Persona
```
Comment (waldirmontoya25) [inline beps/0006-docs-personas-framework-portal/README.md:82]: Hi @aramissennyeydd. I really like this section. Could you clarify a bit more the examples? I thought about the following using your example.

Comment (aramissennyeydd) [reply beps/0006-docs-personas-framework-portal/README.md:82]: I think this looks great! Exactly what I was thinking and I like the separation between role description and Documentation Style. Do you want to update the original bep with these directly? Adding the documentation style solves the other ask I had for what pieces of documentation should go where.

**Review by waldirmontoya25 — APPROVED** [review]

**Pull request merged**